api/indexer/tx_indexer.go

```diff
@@ -28,7 +29,7 @@ var (
 )
 
 func WithIndexer(name string, path string) vm.Option {
-	return func(v *vm.VM) error {
+	return func(v *vm.VM, r json.RawMessage) error {
```

Comment:
Let's change the function signature to use `[]byte` - `json.RawMessage` is just an alias for this anyways

Reply:
done